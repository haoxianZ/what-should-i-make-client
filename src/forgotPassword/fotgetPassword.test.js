import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ForgetPassword from './forgotPassword';

describe(`Forget password component`, () => {
  it('renders without crash', () => {
    const wrapper = shallow(<ForgetPassword />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
}) 