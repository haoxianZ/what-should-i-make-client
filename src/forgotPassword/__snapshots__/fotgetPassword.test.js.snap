// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forget password component renders without crash 1`] = `
<div>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/home-page"
  >
    <header>
      What Should I Make?
    </header>
  </Link>
  <form
    className="forgotPassword"
    onSubmit={[Function]}
  >
    <label
      htmlFor="email"
    >
      Email:
    </label>
    <input
      id="email"
      name="email"
      required={true}
      type="email"
    />
    <button
      type="submit"
    >
      Send Reset Password Email
    </button>
  </form>
</div>
`;
